reporter/src/main/kotlin/model/EvaluatedModel.kt

```diff
@@ -19,7 +19,284 @@
 
 package com.here.ort.reporter.model
 
+import com.fasterxml.jackson.annotation.JsonIdentityInfo
+import com.fasterxml.jackson.annotation.JsonIdentityReference
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.databind.SerializationFeature
+import com.fasterxml.jackson.databind.json.JsonMapper
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
+import com.fasterxml.jackson.databind.introspect.ObjectIdInfo
+import com.fasterxml.jackson.databind.node.ArrayNode
+import com.fasterxml.jackson.databind.node.ObjectNode
+import com.fasterxml.jackson.module.kotlin.registerKotlinModule
+
+import com.here.ort.model.CustomData
+import com.here.ort.model.Identifier
+import com.here.ort.model.LicenseSource
+import com.here.ort.model.OrtIssue
+import com.here.ort.model.OrtResult
+import com.here.ort.model.PROPERTY_NAMING_STRATEGY
+import com.here.ort.model.PackageCurationResult
+import com.here.ort.model.Provenance
+import com.here.ort.model.RemoteArtifact
+import com.here.ort.model.RuleViolation
+import com.here.ort.model.ScannerDetails
+import com.here.ort.model.Severity
+import com.here.ort.model.VcsInfo
+import com.here.ort.model.config.IssueResolution
+import com.here.ort.model.config.LicenseFindingCuration
+import com.here.ort.model.config.PathExclude
+import com.here.ort.model.config.RepositoryConfiguration
+import com.here.ort.model.config.RuleViolationResolution
+import com.here.ort.model.config.ScopeExclude
+import com.here.ort.reporter.Reporter
+import com.here.ort.reporter.ReporterInput
+import com.here.ort.reporter.reporters.WebAppReporter
+import com.here.ort.reporter.utils.IntIdModule
+import com.here.ort.reporter.utils.ZeroBasedIntSequenceGenerator
+import com.here.ort.spdx.SpdxExpression
+
+import java.time.Instant
+import java.util.SortedMap
+import java.util.SortedSet
+
+/**
+ * The [EvaluatedModel] represents the outcome of the evaluation of a [ReporterInput]. This means that all additional
+ * information contained in the [ReporterInput] is applied to the [OrtResult]:
+ *
```

Comment:
Can you say something about the performance of generating this model for large analyzer results, e.g. like the Yarn one we recently ran into performance issues with ?

Reply:
No, because I haven't done performance testing yet.